rero_ils/utils.py

```diff
@@ -127,3 +127,15 @@ def language_iso639_2to1(lang):
         return default_ln
     supported_languages = [v[0] for v in current_i18n.get_languages()]
     return ln if ln in supported_languages else default_ln
+
+
+def language_mapping(lang):
+    """Language mapping.
+
+    :param lang: bibliographic language code
+    :returns: language mapping
+    """
+    mapping = current_app.config.get('RERO_ILS_LANGUAGE_MAPPING', {})
+    if lang in mapping:
+        return mapping[lang]
+    return lang
```

Comment:
We can write:
`return  current_app.config.get('RERO_ILS_LANGUAGE_MAPPING', {}).get(lang, lang)`

Reply:
done